src/main/java/com/kodilla/ecommercee/dto/ProductDto.java

```diff
@@ -15,4 +15,11 @@ public class ProductDto {
     private String description;
     private BigDecimal price;
     private Long groupId;
+
+    public ProductDto(Long productId, String name, String description, BigDecimal price) {
+        this.productId = productId;
+        this.name = name;
+        this.description = description;
+        this.price = price;
+    }
```

Comment:
On postman when I'm adding new product it is added with the groupId set to null (although I was inserting the value in JSON). The same when I'm updating.

Reply:
It's because I have no relations with Group yet.